src/js/containers/aboutTheDataSidebar/AboutTheDataListener.jsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * AboutTheDataListener.jsx
+ * Created by Andrea Blackwell 12/06/2022
+ */
+
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { useLocation } from 'react-router-dom';
+import { omit } from 'lodash';
+
+import * as aboutTheDataActions from 'redux/actions/aboutTheDataSidebar/aboutTheDataActions';
+import { useQueryParams, getQueryParamString } from 'helpers/queryParams';
+
+const AboutTheDataListener = ({
+    history,
+    aboutTheDataSidebar,
+    match,
+    location,
+    showAboutTheData,
+    setAboutTheDataTermFromUrl,
+    Child
+}) => {
+    const { pathname, search } = useLocation();
+    const queryParams = useQueryParams();
+
+    useEffect(() => {
+        // The #fscommand=fstest is used to access the Foresee survey admin panel
+        if (!location.hash || location.hash.indexOf('#fscommand=fstest') > -1) {
+            return;
+        }
+
+        const urlWithNoHash = location.hash.split("#").length > 1
+            ? location.hash.split("#")[1]
+            : '';
+        history.replace(urlWithNoHash);
+    }, [location, history]);
+
+    useEffect(() => {
+        if (search.includes('about-the-data')) {
+            const { "about-the-data": term } = queryParams;
+            showAboutTheData();
+            setAboutTheDataTermFromUrl(term);
+            history.replace({
+                pathname,
+                search: getQueryParamString(omit(queryParams, ['about-the-data']))
+            });
+        }
+    }, [history, aboutTheDataSidebar.display, history.location.search, setAboutTheDataTermFromUrl, search, queryParams, pathname]);
```

Comment:
wasn't sure if we needed the last three dependencies? were they added purposely or by that eslint rule

Reply:
thanks.  fixed